openmetadata-docs/content/deployment/upgrade/docker.md

```diff
@@ -59,7 +73,9 @@ These steps are based on MySQL, but the steps are analogous for Postgres.
     In our case, the backup file was named `openmetadata_202212201528_backup.sql`. You can copy the name from the backup
     command output.
 
-### Add volumes and port mapping
+### 2. Add volumes and port mapping
+
+This step is only required if you had not that yet.
```

Comment:
This step is only required if you had not done that yet.

Reply:
oh :(

Reply:
I need grammarly on IntelliJ

Reply:
thank you